xcell/mappers/mapper_eBOSSQSO.py

```diff
@@ -21,82 +21,96 @@ def __init__(self, config):
         """
         self._get_defaults(config)
 
-        self.cat_data = []
-        self.cat_random = []
-        self.z_arr_dim = config.get('z_arr_dim', 50)
+        self.cats = {'data': [], 'random': []}
```

Comment:
As with decals, I'd make these `None` and `None` to be consistent with the other mappers.